CHANGELOG.md

```diff
@@ -198,12 +198,12 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### Changed
 
-- Modify the `setGigantoConfig` and `gigantoConfig` queries in the GraphQL API
+- Modify the `setConfig` and `config` queries in the GraphQL API
   to read/write the ack transmission count.
-- Modify the `setGigantoConfig` and `gigantoConfig` query in the GraphQL API so
+- Modify the `setConfig` and `config` query in the GraphQL API so
   that the fields that take integers read/write the config file for their
   respective types.
-- Modify the `gigantoConfig` query in the GraphQL API so that config files that
+- Modify the `config` query in the GraphQL API so that config files that
```

Comment:
These lines should remain unchanged, as they accurately reflect the state at the time of the 0.16.0 release. Could you please revert the changes?

Reply:
The change has been completed.